docs/dynamic_batch_size.md

```diff
@@ -0,0 +1,193 @@
+# Dynamic batch size with OpenVINO&trade; Model Server Demultiplexing
+
+## Intoduction
+This documentation presents how to use OpenVINO&trade; Model Server to execute inference requests with dymanic batch size using [Demultiplexing in Directed Acyclic Graph Scheduler](https://github.com/openvinotoolkit/model_server/blob/develop/docs/demultiplexing.md).
+
+With OpenVINO&trade; Model Server Demultiplexing infer request sent from client application can have various batch sizes and changing batch size does not require model reload.
+
+*Note:* When using `demultiply_count` parameters, only one demultiplexer can exist in pipeline.
+
+- The script [grpc_serving_client.py](../example_client/grpc_serving_client.py) runs desired model inference requests for the images in .npy file.
```

Comment:
Example client in python grpc_serving_client.py can be used to request the pipeline. Use `--dag-batch-size-auto` flag to add additional dimension to the input shape which is required for demultiplexing feature.

I would also move client part to the bottom